### DISCUSSION TITLE
Coolify build NextJS Error: Command failed: git --git-dir=/app/.git --work-tree=/app rev-parse HEAD

### DISCUSSION THREAD

**Discussion by IRediTOTO** [coollabsio/coolify#2310] [Q&A]
Hello, I tried to build my nextjs project with Coolify but I have this error
![image](https://github.com/coollabsio/coolify/assets/15695252/44cbc0b1-6a2f-453f-85e5-0763edf189b4)
This is just a normal nextjs not special, It ran well on my old VPS with pm2. But when build with nixpacks I get error 
```
 cmd: 'git --git-dir=/app/.git --work-tree=/app rev-parse HEAD',
 stdout: '',
 stderr: "fatal: not a git repository: '/app/.git'\n"
 }
  ELIFECYCLE  Command failed with exit code 1.
```

**Comment by csshimanshu** [comment]
same issue i got. did you find the solution?

**Reply by IRediTOTO** [reply]
As I remember, this error caused by your project code not about Coolify...

**Reply by csshimanshu** [reply]
Yes this is true! can you also tell me how to fix this?

**Reply by djsisson** [reply]
coolify removes the .git folder after cloning (no idea why)

so if you are trying to run any git commands or get any git info, they will fail, without seeing your code, we can't say what in your codebase is trying to do this.

**Reply by csshimanshu** [reply]
In my case, I am justing Coolify on Hostinger VPS to deploy my NextJS website that I made with AI.

**Reply by djsisson** [reply]
Well the ai has written a git command somewhere or using a package that runs a git command

You can always ask your ai to remove it...

next-build-id is one such package
